resolwe_bio/signals.py

```diff
@@ -0,0 +1,47 @@
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from guardian import shortcuts
+
+from resolwe.flow.models import Data, DescriptorSchema, iterate_schema
+from .models import Sample
+
+
+@receiver(post_save, sender=Data)
+def add_post_save_handler(sender, instance, **kwargs):
+    if kwargs['created'] and instance.process.flow_collection:
```

Comment:
In the future we should replace this parameter with a more general group of parameters for Resolwe extensions. For example, `instance.process.ext.bio.sample`.